src/desdemona/ui/dom.cljs

```diff
@@ -12,6 +12,12 @@
 (def menu-item
   (r/adapt-react-class (aget js/ReactBootstrap "MenuItem")))
 
+(def overlay-trigger
+  (r/adapt-react-class (aget js/ReactBootstrap "OverlayTrigger")))
+
+(def tooltip
+  (r/adapt-react-class (aget js/ReactBootstrap "Tooltip")))
```

Comment:
You could make a function to create these, like "(defn react-class [bootstrap-name] (r/adapt-react-class (aget js/ReactBootstrap bootstrap-name))", and then define them a bit simpler with "(def button (react-class "Button"))". Though to be fair this code is already pretty simple right now.